enterprise/api/v1/views.py

```diff
@@ -456,6 +456,25 @@ class EnterpriseCustomerReportingConfigurationViewSet(EnterpriseReadOnlyModelVie
     ordering_fields = FIELDS
 
 
+class CatalogQueryView(APIView):
+    """
+    View for fetching enterprise catalog query.
+    """
+    authentication_classes = [SessionAuthentication]
```

Comment:
Usually we use `(JwtAuthentication, BearerAuthentication, SessionAuthentication,)` as `authentication_classes`. Don't we need to support other two?

Reply:
We don't really need to support other two since this will only be called via djanog admin tool